lib/applicationModel.js

```diff
@@ -89,54 +90,19 @@ class ApplicationModel {
   }  
 
   setupModelClassMap() {
-    if (!this.modelClassMap) {
-      this.modelClassMap = new Map();
-      this.nameToSchemaMap = new Map();
-      // Register all schemas first, then check the anonymous schemas for duplicates
-      ApplicationModel.asyncapi.allSchemas().forEach((schema, name) => {
-        debugApplicationModel(`setupModelClassMap ${name} type ${schema.type()}`);
-        this.registerSchemaNameToModelClass(schema, name);
-        this.nameToSchemaMap[name] = schema;
-      });
-
-      ApplicationModel.asyncapi.allSchemas().forEach((schema, schemaName) => {
-        debugApplicationModel(`setupModelClassMap anonymous schemas ${schemaName} type ${schema.type()}`);
-        this.registerSchemasInProperties(schema);
-        this.registerSchemasInAllOf(schema);
-      });
-      debugApplicationModel('modelClassMap:');
-      debugApplicationModel(this.modelClassMap);
-    }
-  }
-
-  registerSchemasInProperties(schema) {
```

Comment:
Since we had that bug with $id, we didn't realise that some schemas were being removed from the result of allSchemas(), so this supplemented the inadequacy. Now that the issue is fixed, it's no longer required.